CartPole/__init__.py

```diff
@@ -448,7 +453,7 @@ def Update_Q(self):
                 # in this case slider corresponds already to the power of the motor
                 self.Q = self.slider_value
             else:  # in this case slider gives a target position, lqr regulator
-                self.Q = self.controller.step(self.s_with_noise_and_latency, self.time, {"target_position": self.target_position, "target_equilibrium": self.target_equilibrium})
+                self.Q = self.controller.step(self.s_with_noise_and_latency, self.time, updated_attributes= {"target_position": self.target_position, "target_equilibrium": self.target_equilibrium})
```

Comment:
```suggestion
                self.Q = self.controller.step(self.s_with_noise_and_latency, time=self.time, updated_attributes= {"target_position": self.target_position, "target_equilibrium": self.target_equilibrium})
```
Good catch. Technically, we should also pass the time as keyword-arg, because it is also optional.